src/config/webpack.config.js

```diff
@@ -16,13 +16,17 @@ const fs = require('fs');
 
 const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
 
+const processPath = process.cwd();
+/* Get the root path of a mono-repo process call */
+const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
```

Comment:
Do you need the process path variable? Feels like you can just use process.cwd() on its own after the colon.